src/constants.ts

```diff
@@ -102,6 +102,7 @@ export const TITLE_TIMEOUT = 10000;
 export const TOAST_TIMEOUT = 5000;
 export const VERSION_TIMEOUT = 30000;
 export const SCROLL_TIMEOUT = 100;
+export const DETECT_POSITION_CHANGE_INTERVAL = 100;
```

Comment:
```suggestion
export const DETECT_POSITION_CHANGE_INTERVAL = 500;
```

Reply:
The reason I kept it the same as scroll time out is coz this will be needed when multiple scrolls are also there and we had gotten feedback earlier on scroll timeout not working for some so kept it the same.

Reply:
Right, but unlike scroll handle which is *debounced*, this is an interval, so it's a completely different ball game.

Reply:
oh yeah my bad, I missed that scroll is debounced 👍